### DISCUSSION TITLE
Support cli options for lint command

### DISCUSSION THREAD

**Discussion by stajics** [vercel/next.js#26179] [Ideas]
### Describe the feature you'd like to request

Currently running `next lint --quiet` gives an error `Unknown or unexpected option: --quiet`.

Support passing cli options to lint command.
https://eslint.org/docs/user-guide/command-line-interface#options

### Describe the solution you'd like

Support passing cli options to lint command.
https://eslint.org/docs/user-guide/command-line-interface#options

### Describe alternatives you've considered

/

**Comment by housseindjirdeh** [comment]
Unfortunately not all the ESLint CLI options can be passed as `next lint` uses the [Node API](https://eslint.org/docs/developer-guide/nodejs-api) instead. Most are [supported](https://github.com/vercel/next.js/blob/canary/packages/next/cli/next-lint.ts#L91-L120) however and I've just added support for `--quiet` in this [PR](https://github.com/vercel/next.js/pull/26280).
